website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,137 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
+
+  <CodeBlockConfig hideClipboard>
+
+    $ vault write sys/sync/destinations/$TYPE/$NAME/associations/set \
+    mount="path/to" \
+    secret_name="my-secret-1"
+
+    Error writing data to sys/sync/destinations/$TYPE/$NAME/associations/set: Error making API request.
+
+    URL: PUT http://127.0.0.1:8200/v1/sys/sync/destinations/$TYPE/$NAME/associations/set
+    Code: 403. Errors:
+
+    * permission denied to read the content of the secret my-secret-1 in mount path/to
+
+  </CodeBlockConfig>
+
+This read access verification is only done when creating or updating an association. Once the association is created, revoking
+read access to the policy that was used to sync the secret has no effect.
+
+### Collisions and overwrites
+
+Secrets Sync operates with a last-write-wins strategy. If a secret with the same name already exists at the destination,
+Vault overwrites it when syncing a secret. There are also no automatic mechanisms to prevent a principal with sufficient
+privileges at the destination from overwriting a secret synced by Vault.
+
+To prevent Vault from accidentally overwriting existing secrets, it is recommended to use either the name pattern of
+built-in tag as an extra policy condition on the role used to configure a Vault sync destination. For example, the
+following AWS IAM policy prevents Vault from being able to modify secrets that were not created by a sync operation:
+
+  <CodeBlockConfig hideClipboard>
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "secretsmanager:*",
+          ],
+          "Resource": "*",
+          "Condition": {
+            "StringEquals": {
+              "secretsmanager:ResourceTag/hashicorp:vault": "" # This tag is automatically added by Vault on every synced secrets
+            }
+          }
+        }
+      ]
+    }
+
+  </CodeBlockConfig>
+
+Likewise, is it recommended to add a negative condition on other policies not used by Vault that grant write access to
+secrets to prevent out-of-band overwrites:
+
+  <CodeBlockConfig hideClipboard>
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Deny",
+          "Action": [
+            "secretsmanager:*"
+          ],
+          "Resource": "*",
+          "Condition": {
+            "StringNotEquals": {
+              "secretsmanager:ResourceTag/hashicorp:vault": "" # This tag is automatically added by Vault on every synced secrets
+            }
+          }
+        }
+      ]
+    }
+
+  </CodeBlockConfig>
+
+  <Note>
+  Wildcards are used in the example above for brevity, but it is recommended to use the least privilege principle and restrict
+  the actions and resources to the minimum requirements for each use case.
+  </Note>
+
+This policy condition strategy is also applicable to most other destination types. Please refer to the specific destination's
+access management documentation for more details.
```

Comment:
I added this for now but we have tickets to give examples for [Azure](https://hashicorp.atlassian.net/browse/VAULT-24233) & [GCP](https://hashicorp.atlassian.net/browse/VAULT-24232) as well in their own sections.

Once we have an example for each destination that supports this pattern, we could move the AWS example inline in the AWS SM destination doc as well and only keep the various references on the main page.